src/bundlers/node/genezioRuntimeHandlerGenerator.ts

```diff
@@ -63,6 +44,26 @@ if (!genezioClass) {
             return response;
         }
 
+        try {
+            object = object ?? new genezioClass();
```

Comment:
This can be written as `object ??= new genezioClass();`

Reply:
Thanks!